docs/md_merge.py

```diff
@@ -0,0 +1,357 @@
+import argparse
+import os
+import re
+import shutil
+from argparse import BooleanOptionalAction
+from collections import OrderedDict, defaultdict
```

Comment:
Nit: Why not just import collections and fully specify the five assignments where we use these types?

Reply:
I think that's a question of style. I actually like these imports that way. Or are there any other reasons for your suggestion?

Reply:
I'll resolve this discussion and merge the PR. Please reach out if you still see the need for a change.